introduction.md

```diff
@@ -1,3 +1,10 @@
 Hello! My name is Rudy and I am trying to fulfill my Git path on codeschool.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?